src/components/teams/TeamsFilters.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as teamActions from '../../actions/teamActions';
+import TextInput from '../common/TextInput';
+
+class TeamsFilters extends React.Component {
+  render = () => {
+    return (
+      <TextInput
+        name='team_name'
+        onChange={(event) => this.props.toggleTeam(event.target.value)}
+        className="btn--filter"
+        placeholder="Type name of your team"
+      />
+    )
+  }
+}
+
+function mapStateToProps(state, ownProps) {
+  return {};
```

Comment:
if you dont want to pass anything you can pass `null` as first argument to connect